lib/plugins/create/templates/aws-nodejs-typescript/handler.ts

```diff
@@ -1,13 +1,11 @@
-import { APIGatewayEvent, Callback, Context, Handler } from 'aws-lambda';
+import { APIGatewayProxyHandler } from 'aws-lambda';
 
-export const hello: Handler = (event: APIGatewayEvent, context: Context, cb: Callback) => {
-  const response = {
+export const hello: APIGatewayProxyHandler = async (event, context) => {
```

Comment:
Why did you remove the types from the function parameters?

Reply:
in the current template, it was necessary to explicitly indicate the type of the arguments -- that's because the used insufficient type `Handler`.
now, by storing function to the constant of type `APIGatewayProxyHandler`, the types of `event` and `context` are implicitly inferred. Please see the attached image.

<img width="715" alt="screenshot" src="https://user-images.githubusercontent.com/603523/45680005-cde90100-bb74-11e8-9abb-c49f4360711e.png">

Reply:
hi @uneco 
Thanks for the explanation.

Just saw https://github.com/DefinitelyTyped/DefinitelyTyped/blob/b697ec8a1bb26313b4d56a0af66b9c45c9920cef/types/aws-lambda/index.d.ts#L814 after your explanation and I understood.

:+1: